Part2_search_selection.R

```diff
@@ -144,6 +150,9 @@ bib_data$Volume <- gsub("'", "", bib_data$Volume)
 bib_data$Number <- gsub("'", "", bib_data$Number)
 bib_data$Pages <- gsub("'", "", bib_data$Pages)
 
+# Remove entries without a title
+bib_data <- bib_data[bib_data$Title != "", ]
+
```

Comment:
Hre I removed the entries without a title. That seems to have caused issues later in the BibEntry list.